src/rdf4cpp/BlankNode.cpp

```diff
@@ -116,6 +189,42 @@ void BlankNode::validate(std::string_view v) {
     }
 }
 
+std::strong_ordering BlankNode::order(BlankNode const &other) const noexcept {
+    if (is_inlined()) {
+        if (other.is_inlined()) {
+            return detail_bnode_inlining::from_inlined(handle_.id()) <=> detail_bnode_inlining::from_inlined(other.handle_.id());
+        }
+
+        // this is inlined but other is not, this means that the string of this must be smaller
+        // than the one of other
```

Comment:
doesnt lexicographical order decide on the first non equal char? so `b` > `aa`

Reply:
Ah you are right, good catch